src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -364,6 +363,7 @@ public PravegaStorageDriver(
 			this.transactionMode = eventConfig.boolVal("transaction");
 		} else {
 			this.transactionMode = false;
+			throw new IllegalConfigurationException("ByteStreams are not currently supported. Contact the dev team.");
```

Comment:
Please, add this information into README.md's `Byte Stream` section as well: https://github.com/emc-mongoose/mongoose-storage-driver-pravega/tree/PRAVEGA-101-update-for-0.8-pravega#52-byte-stream-operations